README.md

```diff
@@ -107,7 +107,7 @@ You can find below list of all supported environment variables
 |USE_GITHUB_FAKE_CLIENT | boolean     | "on" / "off"  | false       | use dummy github client        |
 
 
-For further details about availables configuration check <a href="shared/config.ts">shared/config.ts</a>
```

Comment:
Hi @aashish924 

Thanks for the interest on my code, but can you tell me what are you testing exactly?

Regards
Wael